workspaces/src/rpc/tool.rs

```diff
@@ -38,19 +39,23 @@ pub(crate) async fn url_create_account(
     helper_url: Url,
     account_id: AccountId,
     pk: PublicKey,
-) -> anyhow::Result<()> {
-    let helper_addr = helper_url.join("account")?;
+) -> crate::result::Result<()> {
+    let helper_url = helper_url.join("account").unwrap();
 
     // TODO(maybe): need this in near-jsonrpc-client as well:
     let _resp = reqwest::Client::new()
-        .post(helper_addr)
+        .post(helper_url)
         .header("Content-Type", "application/json")
-        .body(serde_json::to_vec(&serde_json::json!({
-            "newAccountId": account_id,
-            "newAccountPublicKey": pk,
-        }))?)
+        .body(
+            serde_json::to_vec(&serde_json::json!({
+                "newAccountId": account_id,
+                "newAccountPublicKey": pk,
+            }))
+            .map_err(BytesError::SerdeError)?,
+        )
         .send()
-        .await?;
+        .await
+        .map_err(|e| Error::Other(Box::new(e)))?;
```

Comment:
Would not this classify as RpcError?

Reply:
nice catch! This definitely belongs with RPC